plugin/src/main/java/org/opennms/velocloud/rest/api/VelocloudRestService.java

```diff
@@ -46,4 +51,24 @@ public interface VelocloudRestService {
     @Path("/partner/{alias}/customers")
     @Produces(value = {MediaType.APPLICATION_JSON})
     List<EnterpriseDTO> getCustomersForMspPartner(@PathParam("alias") String alias) throws VelocloudApiException;
+
+    @GET
+    @Path("/connections/list")
+    @Produces(value = {MediaType.APPLICATION_JSON})
+    List<ConnectionListElementDTO> getConnectionList() throws VelocloudApiException;
+
+    @POST
+    @Path("/connections/{alias}/add")
```

Comment:
```suggestion
    @POST
    @Path("/connections")
```

Reply:
The alias must be either part of the body or this should all be handled by a `PUT` on the URL containing the alias.